frontend/src/metabase/visualizations/echarts/cartesian/option/tooltip.tsx

```diff
@@ -0,0 +1,78 @@
+import type { TooltipOption } from "echarts/types/dist/shared";
+import { renderToString } from "react-dom/server";
+
+import { EChartsTooltip } from "metabase/visualizations/components/ChartTooltip/EChartsTooltip";
+import TooltipStyles from "metabase/visualizations/components/ChartTooltip/EChartsTooltip/EChartsTooltip.module.css";
+import type { ComputedVisualizationSettings } from "metabase/visualizations/types";
+import { getTooltipModel } from "metabase/visualizations/visualizations/CartesianChart/events";
+
+import {
+  GOAL_LINE_SERIES_ID,
+  TIMELINE_EVENT_SERIES_ID,
+} from "../constants/dataset";
+import type { BaseCartesianChartModel, DataKey } from "../model/types";
+
+interface ChartItemTooltip {
+  dataIndex: number;
+  seriesId?: DataKey | null;
+  settings: ComputedVisualizationSettings;
+  chartModel: BaseCartesianChartModel;
+}
+
+const ChartItemTooltip = ({
+  chartModel,
+  settings,
+  dataIndex,
+  seriesId,
+}: ChartItemTooltip) => {
+  if (dataIndex == null || seriesId == null) {
+    return null;
+  }
+
+  const tooltipModel = getTooltipModel(
+    chartModel,
+    settings,
+    dataIndex,
+    seriesId,
+  );
+
+  if (!tooltipModel) {
+    return null;
+  }
+
+  return <EChartsTooltip {...tooltipModel} />;
+};
+
+export const getTooltipOption = (
+  chartModel: BaseCartesianChartModel,
+  settings: ComputedVisualizationSettings,
+): TooltipOption => {
+  return {
+    trigger: "item",
+    appendToBody: true,
+    confine: true,
+    className: TooltipStyles.ChartTooltipRoot,
+    formatter: params => {
+      const isAxisTooltip = Array.isArray(params);
```

Comment:
Sorry if I misunderstood the outcome of the [thread]([url](https://metaboat.slack.com/archives/C064QMXEV9N/p1723483422025159?thread_ts=1723223927.507779&cid=C064QMXEV9N)) and what the code is doing here, but did we decide to not use the axis tooltip for now? If so, do we still need this `isAxisTooltip` variable?